test/python/circuit/library/test_evolution_gate.py

```diff
@@ -34,7 +33,7 @@ class TestEvolutionGate(QiskitTestCase):
     def setUp(self):
         super().setUp()
         # fix random seed for reproducibility (used in QDrift)
-        algorithm_globals.random_seed = 2
+        np.random.seed(2)
```

Comment:
I'm _very_ nervous about setting the global random seed in a non-isolated test like this.  It was bad enough with the `algorithm_globals` object, but this one will affect all other tests, so we'd probably best avoid it.  If we can add a `seed` argument to `QDrift` and update the tests, I think we'll be better off.